### ISSUE TITLE
Can not make request for more tweets after collecting first 21 tweets

### ISSUE THREAD

**Issue by EdwardTang** [tomkdickinson/Twitter-Search-API-Python#3]
Thanks for your assistance! Recently I have run into another issue. With your Python script, Twitter will always response a user profile card within reponse['items_html'] right after first 21 tweets were collected, which makes it impossible to construct the next URL. The HTML of such user profile card is similar to following:

ipdb> print data['items_html']

      <li class="AdaptiveStreamUserGallery AdaptiveSearchTimeline-separationModule js-stream-item" data-component-context="users_module">
  <div class="AdaptiveSearchPage-moduleHeader">
    <h3 class="AdaptiveSearchPage-moduleTitle">
      Accounts
      <a href="/search?f=users&amp;vertical=default&amp;q=from%3AQuantumHydrox" class="AdaptiveSearchPage-moduleLink u-textUserColor js-nav" data-nav="search_filter_users">View all</a>
    </h3>
  </div>

  <div class="Grid Grid--withGutter" role="presentation">
          <div class="Grid-cell">
          <div class="AdaptiveStreamUserGallery-user js-stream-item" data-item-type="user" role="listitem" data-item-id="1015857774" id="stream-item-user-1015857774" data-item-type="user">
                <div class="ProfileCard ProfileCard--wide js-actionable-user"
     data-screen-name="QuantumHydrox"
     data-user-id="1015857774"
     data-feedback-token=""
     data-impression-id=""
     
  >
  <a class="ProfileCard-bg js-nav" href="/QuantumHydrox" tabindex="-1" aria-hidden="true" style="
    background-color: #0084B4;
    ">
  </a>

  <div class="ProfileCard-content">
    <a class="ProfileCard-avatarLink js-nav js-tooltip" href="/QuantumHydrox" title="Quantum Hydrox" tabindex="-1" aria-hidden="true" >
      <img class="ProfileCard-avatarImage js-action-profile-avatar" src="https://pbs.twimg.com/profile_images/3566375811/b5ad5c0d064819457f7c33cf1079d42b_200x200.jpeg" alt="">
    </a>
      <div class="ProfileCardStats">
    <ul class="ProfileCardStats-statList Arrange Arrange--bottom Arrange--equal"><li class="ProfileCardStats-stat Arrange-sizeFit">
        <a class="ProfileCardStats-statLink u-textUserColor u-linkClean u-block js-nav"
            title="257 Tweets"
            href="/QuantumHydrox"
            data-element-term="tweet_stats">
          <span class="ProfileCardStats-statLabel u-block">Tweets</span>
          <span class="ProfileCardStats-statValue" data-is-compact="false">257</span>
        </a>
      </li><li class="ProfileCardStats-stat Arrange-sizeFit">
          <a class="ProfileCardStats-statLink u-textUserColor u-linkClean u-block js-nav"
              title="94 Following"
              href="/QuantumHydrox/following"
              data-element-term="following_stats">
            <span class="ProfileCardStats-statLabel u-block">Following</span>
            <span class="ProfileCardStats-statValue" data-is-compact="false">94</span>
          </a>
        </li><li class="ProfileCardStats-stat Arrange-sizeFit">
          <a class="ProfileCardStats-statLink u-textUserColor u-linkClean u-block js-nav"
              title="31 Followers"
              href="/QuantumHydrox/followers"
              data-element-term="follower_stats">
            <span class="ProfileCardStats-statLabel u-block">Followers</span>
            <span class="ProfileCardStats-statValue" data-is-compact="false">31</span>
          </a>
        </li>
    </ul>
  </div>

      <div class="ProfileCard-actions">
        <div class="ProfileCard-userActions with-rightCaret js-userActions">
          <div class="UserActions   u-textLeft" >
    <div class="user-actions btn-group not-following " data-user-id="1015857774"
        data-screen-name="QuantumHydrox" data-name="Quantum Hydrox" data-protected="false">
      <span class="UserActions-moreActions u-inlineBlock">
          <button type="button" class="js-tooltip unmute-button btn small plain-btn" title="Unmute @QuantumHydrox" data-placement="top">
            <span class="Icon Icon--muted Icon--medium"><span class="visuallyhidden">Unmute @QuantumHydrox</span></span>
          </button><button type="button" class="first-load js-tooltip mute-button btn small plain-btn" title="Mute @QuantumHydrox" data-placement="top">
            <span class="Icon Icon--unmuted Icon--medium"><span class="visuallyhidden">Mute @QuantumHydrox</span></span>
          </button>

      </span><button class="user-actions-follow-button js-follow-btn follow-button btn" type="button">
  <span class="button-text follow-text">
     <span class="Icon Icon--follow"></span> Follow 
    
  </span>
  <span class="button-text following-text">
     Following 
    
  </span>
  <span class="button-text unfollow-text">
     Unfollow 
    
  </span>
  <span class="button-text blocked-text">Blocked</span>
  <span class="button-text unblock-text">Unblock</span>
  <span class="button-text pending-text">Pending</span>
  <span class="button-text cancel-text">Cancel</span>
</button>

    </div>
</div>

        </div>
      </div>

    <div class="ProfileCard-userFields">
      <div class="ProfileNameTruncated">
  <div class="u-textTruncate u-inlineBlock">
    <a class="ProfileNameTruncated-link u-textInheritColor js-nav js-action-profile-name" href="/QuantumHydrox"  data-aria-label-part>
      Quantum Hydrox
    </a>
  </div>
</div>
      
      <span class="ProfileCard-screenname u-dir" dir="ltr">
        <a href="/QuantumHydrox" class="ProfileCard-screennameLink u-linkComplex js-nav" data-aria-label-part >
          @<span class="u-linkComplex-target">QuantumHydrox</span>
        </a>
      </span>
      <p class="ProfileCard-bio u-dir js-ellipsis"
        
        dir="ltr" data-aria-label-part>Luxe Skin Smoothing System</p>
          <p class="ProfileCard-locationAndUrl">  <a class="u-textUserColor" target="_blank" rel="me nofollow" href="http://t.co/IpYNPX6jdV" title="http://www.quantumhydrox.com/rejuvenation/">
    quantumhydrox.com/rejuvenation/
  </a>

</p>

    </div>
  </div>
</div>

          </div>
        </div>
  </div>
</li>

And if you open the search page in browser,  the profile card looks like this:

![image](https://cloud.githubusercontent.com/assets/3278807/11081345/62318b36-87e1-11e5-93a1-0d50e1bfabce.png)

Please advise!

**Comment by tomkdickinson** [comment]
Ok, I think I've reproduced the issue. It's caused when scraping specific users using the "from:userName" tag. I'll take a look at it later today and see if I can come up with a fix.

**Comment by tomkdickinson** [comment]
I've added a user-agent header when opening a request to Twitter. Seems to have fixed the problem.

**Comment by EdwardTang** [comment]
Thanks! I fixed the problem just then.
